src/App.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <h1>MEMORY GAMME</h1>
+  <button @click="openFullScreen" class="fullScreenButton"></button>
```

Comment:
klasy nie mają cammel case'a tylko lepiej jest ich używać z dashem
```suggestion
  <button @click="openFullScreen" class="fullscreen-button"></button>
```